### PR TITLE
[Specs Change] An optimization in CONSTBUFFER_CreateFromOffsetAndSize

### PR DESCRIPTION
**Pull request by RohitG28**
Specification addition in CONSTBUFFER_CreateFromOffsetAndSize.

### CODE REVIEW

File: devdoc/constbuffer_requirements.md [hunk 1]
```diff
@@ -172,6 +172,8 @@ Given an existing `handle` `CONSTBUFFER_CreateFromOffsetAndSize` creates another
 
 **SRS_CONSTBUFFER_02_027: [** If `offset` + `size` exceed `handles`'s size then `CONSTBUFFER_CreateFromOffsetAndSize` shall fail and return `NULL`. **]**
 
+If `offset` is 0 and `size` is equal to `handle`'s size then `CONSTBUFFER_CreateFromOffsetAndSize` shall return `handle`.
```
Comment (dcristoloveanu) [inline devdoc/constbuffer_requirements.md:175]: >`CONSTBUFFER_CreateFromOffsetAndSize` shall return `handle`. [](http://example.com/codeflow?start=61&length=60)

I think we also have to specify that we increment the reference count.

**Review by dcristoloveanu — CHANGES_REQUESTED** [review]
:clock1:

File: devdoc/constbuffer_requirements.md (hunk 1 above)
Comment (RohitG28) [reply devdoc/constbuffer_requirements.md]: Done

**Review by dcristoloveanu — APPROVED** [review]
:shipit:

**Pull request merged**